### PR TITLE
Always let grunt know when grunt-sass is done

### LINKED ISSUE #287: Since v3 grunt-contrib-watch fails to recover after grunt-sass recovers from an error
3.0.1 seems correctly pass sass errors to grunt indeed, but then, after amending the error, grunt-contrib-watch only logs the file change and stops working.

### PR DESCRIPTION
**Pull request by bengosney**
Fixes #287 where grunt-watch would not work after there had been an error

### CODE REVIEW

**Comment by bengosney** [comment]
The failure is with existing code

**Comment by Hlsgs** [comment]
@bengosney thank you! hope this gets merged soon.

File: tasks/sass.js [hunk 1]
```diff
@@ -35,8 +35,6 @@ module.exports = grunt => {
 					grunt.file.write(filePath, result.map);
 				}
 			}));
-
-			done();
-		})().catch(grunt.fatal);
+		})().catch(grunt.fatal).finally(done);
```
Comment (sindresorhus) [inline tasks/sass.js]: `Promise#finally` is not available in Node.js 8, which this plugin supports.

**Comment by Hlsgs** [comment]
All good. Thanks, @bengosney !

**Pull request merged**